Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Rifles/rifles.yml

```
@@ -417,14 +416,13 @@
           tags:
           - MagazineAR18
-          - MagazineRifle
       gun_chamber:
         name: Chamber
-        startingItem: CartridgeRifle  # 556
+        startingItem: CartridgeHeavyRifleNT  # 7.62х51
         priority: 1
         whitelist:
           tags:
-          - CartridgeRifle  # 556
+          - CartridgeHeavyRifleNT  #7.62х51 NT
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 12 but found 10

(indentation)